vm/hugepage/libhugetlbfs/Makefile

```diff
@@ -40,11 +40,12 @@ PATCH_DIR=patches
 ifndef SYSENV
 	SYSENV := $(shell uname -i)
 endif
-PYTHON_VERSION	:=$(shell which python3 &>/dev/null && echo python3)
+ARCH         := $(SYSENV)
+OS_MAJOR_RELEASE := $(shell grep -Go 'release [0-9]\+' /etc/redhat-release | sed 's/release //')
```

Comment:
Shouldn't we consider compatible with Fedora and RHEL9? AFAIK, most of CKI jobs testing on Fedora+cki-kernel.